internal/builder/lifecycle.go

```diff
@@ -15,9 +15,9 @@ import (
 )
 
 const (
-	DefaultLifecycleVersion    = "0.5.0"
+	DefaultLifecycleVersion    = "0.6.0"
 	DefaultBuildpackAPIVersion = "0.2"
-	DefaultPlatformAPIVersion  = "0.1"
+	DefaultPlatformAPIVersion  = "0.2"
```

Comment:
I'm not finding any reference to this outside of `acceptance_test.go`. I think we should move it there and not make it exported. This was probably was left from when we defaulted when the lifecycle didn't provide that information. We now have a hard requirement so this should not be necessary.